.github/workflows/pkgdown.yml

```diff
@@ -0,0 +1,76 @@
+on:
+  workflow_call:
+    inputs:
+      RCLR_VERSION:
+        required: true
+        type: 'string'
+      extra-packages:
+        required: false
+        type: 'string'
+
+name: pkgdown
+
+jobs:
+  build-site:
+    runs-on: windows-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install PK-Sim
+        uses: Open-Systems-Pharmacology/Workflows/.github/actions/install-PKSIM
+
+      - uses: r-lib/actions/setup-pandoc@v2
+
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          use-public-rspm: true
+
+      - name: get rClr
+        uses: Open-Systems-Pharmacology/Workflows/.github/actions/get-rClr
+        with:
+          RCLR_VERSION: ${{ inputs.RCLR_VERSION }}
+
+      - name: Install package dependencies
+        uses: Open-Systems-Pharmacology/Workflows/.github/actions/install-package-deps
+        with:
+          RCLR_VERSION: ${{ inputs.RCLR_VERSION }}
+          extra-packages: ${{ inputs.extra-packages }}
+
+      - name: Build site
+        run: pkgdown::build_site_github_pages(new_process = FALSE, install = FALSE)
+        shell: Rscript {0}
+
+      - name: Upload Artifacts 🔺 # The project is then uploaded as an artifact named 'site'.
+        uses: actions/upload-artifact@v1
+        with:
+          name: docs
+          path: docs
+
+
+  upload-to-branch:
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    permissions:
+      contents: write
+    needs: [pkgdown] # The second job must depend on the first one to complete before running and uses ubuntu-latest instead of windows.
```

Comment:
Problem?
![grafik](https://github.com/Open-Systems-Pharmacology/Workflows/assets/25061876/cb5d9912-ad05-479b-9efd-a06262453567)

Reply:
Good catch, the `build-site` job was previously named `pkgdown` and I changed it without updating this reference.

Fixed in latest commit.